spec/features/admin/sort_view_in_organisation_list_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe 'sorting the columns in the organisation list' do
+  context 'when admin views the names of organisations' do
+    let!(:organisation_1) { create(:organisation, name: "ABC", created_at: Date.yesterday) }
```

Comment:
I've found `let!` to be less clear than the alternatives, most of the time.

Since we don't actually reference `organisation_1` as a variable in the specs, you could move creation the organisations to the `before` block:

```ruby
before do
  create(:organisation, name: "ABC", created_at: Date.yesterday)
  create(:organisation, name: "DEF", created_at: 1.month.ago)
  create(:organisation, name: "XYZ", created_at: 2.years.from_now)

  sign_in_user user
  visit root_path
end
```

I think that's clearer than having two separate blocks as setup.